components/concept/engine/src/main/java/mozilla/components/concept/engine/webnotifications/WebNotification.kt

```diff
@@ -37,4 +38,5 @@ data class WebNotification(
     val engineNotification: Parcelable,
     val timestamp: Long = System.currentTimeMillis(),
     val triggeredByWebExtension: Boolean = false,
+    val silent: Boolean = true,
```

Comment:
Huh, thanks for adding this; didn't realize that it wasn't passed in here already.